Source/Csla.Channels.RabbitMq/RabbitMqChannelExtensions.cs

```diff
@@ -58,7 +58,7 @@ public static DataPortalServerOptions UseRabbitMqPortal(this DataPortalServerOpt
       options?.Invoke(portalOptions);
 
       config.Services.AddScoped(_ => portalOptions);
-      config.Services.AddTransient<RabbitMqPortalFactory>();
+      config.Services.AddTransient<IRabbitMqPortalFactory, RabbitMqPortalFactory>();
```

Comment:
You now have to use the new Type property for registration so the configuration is respected.

Reply:
Doh!